sdcm/nemesis.py

```diff
@@ -3387,17 +3387,20 @@ def disrupt(self):
         self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list)
 
 
+cloud_limited_chaos_monkey = ['disrupt_nodetool_cleanup',
```

Comment:
Please, use uppercase for constants. Also, think about using a tuple instead of a list to prevent accidental modification of this constant.

Reply:
fixed upper case. tuple sounds like too much overhead since all functions are expecting lists.